pkg/server/api/routes/instances.go

```diff
@@ -65,11 +65,12 @@ func (i Instances) Create(w http.ResponseWriter, r *http.Request) error {
 		return nil
 	}
 
-	instance := models.NewInstance(imageID, email)
+	accessToken := r.Context().Value(middleware.AccessTokenKey).(string)
```

Comment:
It will be the default value if it doesn't exist. So an empty string. This is what we want for the shared secret authentication. Is that what you mean?